src/features/usersData.jsx

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const initialState = {
```

Comment:
this json data you can create it folder like data or other name you want not put json to features